djchoices/tests/test_choices.py

```diff
@@ -65,6 +67,7 @@ def test_class_values(self):
         self.assertEqual(SubClass1.values[SubClass1.Item_1], "Item 1")
         self.assertEqual(SubClass1.values[SubClass1.Item_4], "Item 4")
         self.assertEqual(SubClass1.values[SubClass1.Item_5], "Item 5")
+        self.assertIsInstance(SubClass1.values, OrderedDict)
```

Comment:
IIRC my first instinct here was to change the test to compare values at once instead of doing a ton of one-element comparisons, but I didn’t want to go against the style of other tests, so I made the shortest meaningful change instead.  I can see how your suggestion is about the behaviour more than the implementation; I can easily make that change.